source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -102,10 +108,12 @@ The following examples use the ``theaters`` collection of the
 ``sample_mflix`` sample database available in MongoDB Atlas, with a
 ``2dsphere`` index on the ``location.geo`` field.
 
+.. include:: /includes/sample-data-note-plural.rst
```

Comment:
I agree with the idea of making this information more clear than what's presented in the prior paragraph ("sample database available in MongoDB Atlas").

However, I think we shouldn't repeat that information twice. One potential solution:
Lead with the explanation contained in the note of how to get the sample data and then start a new paragraph that describes what the examples use from that database.

Reply:
Thanks for the note. I agree that repeating the "uses the sample Atlas Dataset" information twice is not productive. I have removed the note, and replaced it with the following paragraph linking to the quickstart:

>  If you want to test the examples using the sample dataset, you can learn how to
set up a free-tier Atlas cluster with the ``sample_mflix``
database by following our :doc:`quickstart guide </quick-start>`.

Reply:
Thanks for the note. I agree that the repetition of "use the Atlas Sample Dataset" is not productive. I like your proposed solution! I agree that it is productive to go from "There is a sample dataset that you can install this way" to "here is what is used from the sample dataset". I have removed the included note as it no longer feels necessary. Let me know if you have any notes. 

Fix: https://github.com/mongodb/docs-node/pull/167/commits/f0749a80d608dc69f6b5905a57367ef58d72cefb